jsons/Buildings.json

```diff
@@ -78,8 +80,9 @@
         "gold": 2,
         "specialistSlots": {"Merchant": 1},
         "hurryCostModifier": 15,
-        "uniques": ["[+5]% [Production] [in this city]",
-            "[+25]% [Gold] [in this city]","Requires a [Market] [in this city]"],
+	  "percentStatBonus": {"gold": 25},
+	  "uniques": ["[+5]% [Production] [in this city]"],
+	  "requiredBuilding": "Market",

```

Comment:
why this change tho? requiredBuilding and percentStatBonus are an old way of doing it.

Reply:
See here my proof: https://github.com/yairm210/Unciv/blob/master/android/assets/jsons/Civ%20V%20-%20Gods%20%26%20Kings/Buildings.json (last accessed 23/03/2022 16:14).